📰 Write blog post recruiting a new board member (#61)
DEFNA/defna.org: _posts/2024-02-29-defna-board-member-recruitment.md

---
layout: post
title:  "Join DEFNA! Board Member Recruitment"
date: 2024-02-29 12:30:00
categories: ""
author_name : DEFNA
author_url : /about
author_avatar: defna
show_avatar : true
read_time : ""
feature_image: 2024-02-29
show_related_posts: false
permalink: /announcements/2024/02/29/defna-board-member-recruitment/
---

> Django Events Foundation North America (DEFNA) is looking for another board member. We have a nine-member board; one of our board members is stepping down as soon as a replacement is confirmmed. We are looking for a new board member who is interested in growing the DjangoCon US community.

Reviewer: ```suggestion
Django Events Foundation North America (DEFNA) is looking for another board member. We have a nine-member board; one of our board members is stepping down as soon as a replacement is confirmed. We are looking for a new board member who is interested in growing the DjangoCon US community.
```

> 

Reviewer: ```suggestion

Currently we don't have a Corporate Treasurer to manage our finances. We're considering hiring an accounting firm to do most of the finance work. This means the new Treasurer would focus more on overseeing our finances instead of doing all the data entry. If you're passionate about DjangoCon US and have some experience or interest in managing finances, we'd love to hear from you.

```

---
layout: post
title:  "Join DEFNA! Board Member Recruitment"

> date: 2024-02-29 12:30:00

Reviewer: ```suggestion
date: 2024-03-26 12:30:00
```

categories: ""
author_name : DEFNA
author_url : /about
author_avatar: defna
show_avatar : true
read_time : ""

> feature_image: 2024-02-29

Reviewer: ```suggestion
feature_image: 2024-03-26
```

show_related_posts: false

> permalink: /announcements/2024/02/29/defna-board-member-recruitment/

Reviewer: ```suggestion
permalink: /announcements/2024/03/26/defna-board-member-recruitment/
```

---

> Django Events Foundation North America (DEFNA) is looking for another board member. We have a nine-member board; one of our board members is stepping down as soon as a replacement is confirmed. We are looking for a new board member who is interested in growing the DjangoCon US community.

Author: ```suggestion
Django Events Foundation North America (DEFNA) is looking for another board member. We have a nine-member board; one of our board members will be stepping down as soon as a replacement is confirmed. We are looking for a new board member who is interested in growing the DjangoCon US community.
```

Currently we don't have a Corporate Treasurer to manage our finances. We're considering hiring an accounting firm to do most of the finance work. This means the new Treasurer would focus more on overseeing our finances instead of doing all the data entry. If you're passionate about DjangoCon US and have some experience or interest in managing finances, please highlight that!

Board membership takes, on average, about five hours per month performing the following functions:

* Attend the monthly board meeting
* Participate in the grant applications review process
* Use DEFNA's social media channels to communicate about DEFNA
* Come up with new and clever ways for DEFNA to fulfill our mission

If you'd like to be considered for the board, drop us a note at hello@defna.org, and let us know:

* What interests you about being on the board
* Your current/prior community involvement (it's okay if you're new—we need everyone to participate!)
* Any particular interests you have (grants committee, corporate filings, corporate treasurer, etc.)
* Any ideas you have for expanding DEFNA's reach in the OSS community
* Anything else you'd like us to know about you as a potential board member

> Please email your information to us by March 15 at 6:00 PM Pacific Time. We'll contact you if we need more details. We'll make our decision and reply to everyone by end-of-day April 4th.

Reviewer 2: We need to update our deadline dates here.
```suggestion
Please email your information to us by April 2 at 6:00 PM Pacific Time. We'll contact you if we need more details. We'll make our decision and reply to everyone by end-of-day April 3th.
```

> Please email your information to us by March 15 at 6:00 PM Pacific Time. We'll contact you if we need more details. We'll make our decision and reply to everyone by end-of-day April 4th.

Reviewer: ```suggestion
Please email your information to us by April 9th at 6:00 PM Pacific Time. We'll contact you if we need more details. We'll make our decision and reply to everyone by end-of-day April 25th.
```

Reviewer: @PeterGrand we should give folks two weeks.
